src/EditorFeatures/Core/Implementation/InlineRename/InlineRenameSession.cs

```diff
@@ -805,12 +806,12 @@ private void ApplyRename(Solution newSolution, IWaitContext waitContext)
 
                 if (newDocument.SupportsSyntaxTree)
                 {
-                    var root = newDocument.GetSyntaxRootSynchronously(waitContext.CancellationToken);
+                    var root = newDocument.GetSyntaxRootSynchronously(CancellationToken.None);
```

Comment:
i dont' understand why this cahnge.  you already did the `.AddScope(allowCancellation: false)` above, so this cancellation token can't ever trigger.  So it would be fine to keep passing it in.  That is much clearer to me instead of having to reason about code that has both the context cancellation token, and the .None.

Reply:
Ah, yeah, switching back. I was very suspicious of code that was being given a cancellation token that wasn't going to be set, but at the same time would have probably malfunctioned if it actually _was_ given a cancellation token. I guess this is just recomputing a Solution snapshot, so would have been safe.

Reply:
Oh right didn't follow back up on this. So I can:

1. Leave the code as is. I can add more comments.
2. Add the cancellation tokens back, but recognize they're not actually going to be triggered, and if ever they are, the code will be broken.
3. Actually try to fix function to handle cancellation, but that'll potentially require a fair bit of refactoring of this and may only break something further.

Thoughts @CyrusNajmabadi? I'm not against just fixing this, I just worry about what I might break.

Reply:
looking.

Reply:
add more comments and i'm fine. thanks.